### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by envygeeks**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,13 +20,13 @@ if RUBY_PLATFORM =~ /cygwin/ || RUBY_VERSION.start_with?("2.2")
 end
 
 gem 'rake', '~> 10.1'
-gem 'rdoc', '~> 3.11'
+gem 'rdoc', '~> 4.2'
```
Comment (parkr) [inline Gemfile]: I don't think we use `rdoc` anymore. Is this still necessary?

**Comment by parkr** [comment]
Pulling down to check everything. `mime-types` isn't tested so bump could cause issues.

**Comment by parkr** [comment]
Somehow, the update from cucumber 1.3 to 2.x didn't break anything.

**Comment by envygeeks** [comment]
Yeah that confused me too.  I thought for sure Cucumber would break but it didn't.

**Pull request merged**